src/app/pages/dashboard/components/widget-sys-info/widget-sys-info.component.ts

```diff
@@ -284,15 +284,20 @@ export class WidgetSysInfoComponent extends WidgetComponent implements OnInit {
     this.ws.call('update.check_available').pipe(
       take(1),
       untilDestroyed(this),
-    ).subscribe((update) => {
-      if (update.status !== SystemUpdateStatus.Available) {
-        this.updateAvailable = false;
-        sessionStorage.updateAvailable = 'false';
-        return;
-      }
+    ).subscribe({
+      next: (update) => {
+        if (update.status !== SystemUpdateStatus.Available) {
+          this.updateAvailable = false;
+          sessionStorage.updateAvailable = 'false';
+          return;
+        }
 
-      this.updateAvailable = true;
-      sessionStorage.updateAvailable = 'true';
+        this.updateAvailable = true;
+        sessionStorage.updateAvailable = 'true';
+      },
+      error: (err) => {
```

Comment:
Error handler can be removed, uncaught errors will be logged to sentry automatically.